server/views/applications/add_key.njk

```diff
@@ -66,51 +64,50 @@
+  <div class="govuk-form-group{{' govuk-form-group--error' if this_errors.restrictedIp is not undefined }}" id='restricted-ips' {{'' if  this_data.body.keyType === 'rest' or this_data.body.keyType === 'stream' else 'style=display:none;' }}>
     <label class="govuk-label" for="more-detail-ip">
       Restricted IPs
     </label>
 
     <span id="more-detail-hint-ip" class="govuk-hint">
       Separate each IP with a comma.
     </span>
-
-    <textarea class="govuk-textarea{{ ' govuk-input--error' if this_errors.restrictedIp is not undefined }}" id="more-detail-ip" name="restrictedIps" rows="5" aria-describedby="more-detail-hint-ip">{{ this_data.body.restrictedIps }}</textarea>
     {% if this_errors.restrictedIp is not undefined %}
       <span id="err-id-terms" class="govuk-error-message">
         <span class="govuk-visually-hidden">Error:</span> {{ this_errors.restrictedIp.inline }}
       </span>
     {% endif %}
+    <textarea class="govuk-textarea{{ ' govuk-input--error' if this_errors.restrictedIp is not undefined }}" id="more-detail-ip" name="restrictedIps" rows="5" aria-describedby="more-detail-hint-ip">{{ this_data.body.restrictedIps }}</textarea>
   </div>
-  <div class="govuk-form-group" id='js-domains' {{'' if  this_data.body.keyType === 'rest' else 'style=display:none;' }}>
+  <div class="govuk-form-group{{' govuk-form-group--error' if this_errors.javaScriptDomain is not undefined }}" id='js-domains' {{'' if  this_data.body.keyType === 'rest' else 'style=display:none;' }}>
     <label class="govuk-label" for="more-detail-domain">
       JavaScript domains
     </label>
 
     <span id="more-detail-hint-domain" class="govuk-hint">
       Separate each domain with a comma.
     </span>
-    <textarea class="govuk-textarea{{ ' govuk-input--error' if this_errors.javaScriptDomain is not undefined }}" id="more-detail-domain" name="javaScriptDomains" rows="5" aria-describedby="more-detail-hint-domain">{{ this_data.body.javaScriptDomains }}</textarea>
     {% if this_errors.javaScriptDomain is not undefined %}
       <span id="err-id-terms" class="govuk-error-message">
         <span class="govuk-visually-hidden">Error:</span> {{ this_errors.javaScriptDomain.inline }}
       </span>
     {% endif %}
+    <textarea class="govuk-textarea{{ ' govuk-input--error' if this_errors.javaScriptDomain is not undefined }}" id="more-detail-domain" name="javaScriptDomains" rows="5" aria-describedby="more-detail-hint-domain">{{ this_data.body.javaScriptDomains }}</textarea>
   </div>
-  <div class="govuk-form-group" id='redirect-uris' {{'' if  this_data.body.keyType === 'web' else 'style=display:none;' }}>
+  <div class="govuk-form-group{{' govuk-form-group--error' if this_errors.redirectURIs is not undefined }}" id='redirect-uris' {{'' if  this_data.body.keyType === 'web' else 'style=display:none;' }}>
```

Comment:
happens on a few locations, no space after the {{ opens.

Reply:
changed

Reply:
changed some more

Reply:
Doesn't look like all have been changed yet